syntax = "proto3";

package pneus;

// O servi√ßo que o SSACP vai oferecer
service PneuStorage {
  rpc EnviarDadosPneu (DadosPneu) returns (RespostaArmazenamento) {}
}

// A estrutura dos dados
message DadosPneu {
  int32 carro_id = 1;
  float pressao = 2;
  float temperatura = 3;
  float desgaste = 4;
  string timestamp = 5;
}

// A resposta
message RespostaArmazenamento {
  bool sucesso = 1;
  string mensagem = 2;
}